apps/backend/src/lib/rds.ts

```diff
@@ -22,6 +31,106 @@ export class RDS {
             .then((xs) => xs[0]?.id ?? null);
     }
 
+    /**
```

Comment:
This class is getting very bloated and should be broken up, but you can just leave it as is and clean up later.